packages/request-client.js/test/in-memory-request.test.ts

```
@@ -1,9 +1,27 @@
+
+class MyCustomDataAccess extends CombinedDataAccess {
+  constructor() {
+    const pendingStore = new PendingStore();
+    super(
+      new DataAccessRead({} as any, { network: 'mock', pendingStore }),
+      new NoPersistDataWrite(),
+    );
+  }
+}
+{
+}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)